src/js/api/apiBase.js

```diff
@@ -1,2 +1,2 @@
-export const apiBase = "https://whatver.com/api/posts";
```

Comment:
We shouldn't have the linting/formatting changes in this PR. A separate PR must be made for these because they clutter up the files changed section and make it harder to locate the configuration changes themselves.

Reply:
To fix the issue should create a new branch for your formatting changes?

Reply:
Yeah you will need to remove them from this branch and place them into a new branch. To do this you can either revert or cherry-pick a commit.

Reply:
I appreciate your feedback, coach.
I will work on making the necessary improvements.

Reply:
I have removed the formatting changes and placed them in a separate branch 'format-changes'.

I have tried to add you to the new PR, but there is still an issue with adding you in. 

PR: link is below
https://github.com/NoroffFEU/social-media-client/pull/147 

Please review the changes and let me know if there are any additional modifications that need to be made. 



Thank you